cmd/rokka/cli/sourceimages.go

```diff
@@ -165,6 +173,24 @@ var sourceImagesDeleteCmd = &cobra.Command{
 	Run: run(deleteSourceImage, "Successfully deleted source image.\n"),
 }
 
+var sourceImagesRestoreCmd = &cobra.Command{
+	Use:                   "restore [org] [hash]",
+	Short:                 "Restore a source image by hash",
+	Args:                  cobra.ExactArgs(2),
+	Aliases:               []string{"r"},
+	DisableFlagsInUseLine: true,
+	Run: run(restoreSourceImage, "Successfully restored source image.\n"),
+}
+
+var sourceImagesCopyCmd = &cobra.Command{
+	Use:                   "copy [source_org] [hash] [destination_org]",
```

Comment:
please use `camelCase` also for the doc placeholders. It doesn't really matter, just to be consistent.